src/components/label/label.ts

```diff
@@ -10,7 +10,8 @@ class LabelComponent implements WebComponent {
   render(): Element {
     this.label = createElementFromString(`${this.original ?
       `<div class="label">${this.text}</div>` :
-      `<input type="text" class="label label--alternative" placeholder="${this.text}">`}`) as HTMLElement;
+      `<input type="text" class="label label--alternative" placeholder="${this.text}" 
+      required maxlength="15" pattern="[A-Za-z0-9]+">`}`) as HTMLElement;
```

Comment:
Zmieniłbym na maxlength="9", jak sprawdzałem to 9/10 znaków się mieści, więc lepiej dać tą dolną granicę